addons/point_of_sale/tools/posbox/overwrite_before_init/etc/init_posbox_image.sh

```diff
@@ -237,15 +237,17 @@ systemctl disable hostapd.service
 systemctl disable cups-browsed.service
 systemctl enable odoo.service
 
-# disable overscan in /boot/config.txt, we can't use
-# overwrite_after_init because it's on a different device
-# (/dev/mmcblk0p1) and we don't mount that afterwards.
-# This option disables any black strips around the screen
-# cf: https://www.raspberrypi.org/documentation/configuration/raspi-config.md
-echo "disable_overscan=1" >> /boot/config.txt
-
-# Use the fkms driver instead of the legacy one (RPI3 requires this)
-sed -i '/dtoverlay/c\dtoverlay=vc4-fkms-v3d' /boot/config.txt
+# Use the following video drivers:
+#  Pi 3 - default (legacy)
+#  Pi 4/5 - KMS
+sed -i '/dtoverlay/c\' /boot/config.txt
+echo "
+[pi4]
+dtoverlay=vc4-kms-v3d
+[pi5]
+dtoverlay=vc4-kms-v3d
```

Comment:
this fixes the printers issue on RPI4/5 without breaking RPI3 screens, but what about cups issue on RPI3s ?

Reply:
I've just tested both with and without a display on a Pi 3, and it seems to work fine, `lpinfo -m` returns and printers are detected.
I'll do the same testing on Pi 4 and Pi 5 just to make sure the fix is working correctly.

Reply:
Ok great!

Reply:
It turns out just using the newest version of Raspberry Pi OS fixes this problem without needing any config changes (see PR #187206). So I think we can probably close this PR.